test/phylogenies.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * Test phylogenies.
+ */
+
+/* eslint-env mocha */
+
+// Load d3 as a global variable so it can be accessed by both phylotree.js (which
+// needs to add additional objects to it) and phyx (which needs to call it).
+global.d3 = require('d3');
+
+if (!Object.prototype.hasOwnProperty.call(global.d3, 'layout')) {
+  global.d3.layout = {};
+}
+require('../lib/phylotree.js/phylotree.js');
+
+const chai = require('chai');
+const phyx = require('../js/phyx');
+
+const { assert } = chai;
```

Comment:
Is there a reason to not simply state `const assert = chai;`? You're changing the type from object to function by what you wrote.

Reply:
`const { assert } = chai;` is identical to `const assert = chai.assert;`, but eslint prefers object destructuring, which is why I use the former syntax. I could just use `chai.assert.isOk(...)` everywhere, but I think `assert` is pretty unambiguous.

Reply:
I see what you mean now, I missed that. This means we require at least node 6.x, right?

Reply:
Yup, [that's right](https://node.green/#ES2015-syntax-destructuring--assignment-with-objects)! I've filed a bug so I update `package.json` to require that (phyloref/curation-tool#76).